static/docs/commands-reference/run.md

```diff
@@ -12,24 +12,27 @@ usage: dvc run [-h] [-q | -v] [-d DEPS] [-o OUTS] [-O OUTS_NO_CACHE]
                [--ignore-build-cache] [--remove-outs] [--no-commit]
                [--outs-persist OUTS_PERSIST]
                [--outs-persist-no-cache OUTS_PERSIST_NO_CACHE]
-               command
+               ...
```

Comment:
I would keep the `command`. Especially considering that we have it two-three lines below. No matter what we have in the DVC itself

Reply:
But the actual usage output of `dvc run` has `...`, not `command` in that line. My suggestion is to improve the description to explain the ellipsis symbol, see 988dc79. Lmk if you'd like me to revert that last change and just change `...` for `command` though, as you prefer )

Reply:
I would keep command ... it's easier to understand. We don't need to be precise with DVC. I would try to fix it on the DVC side as well.

Reply:
I tried fixing it on the core repo but it's how argparse outsputs it automtically since the argument takes all remanent command-line inputs. (It's an option in argparse and `dvc run` is the only one that uses it.)

Anyway, I added `command` in the doc and remove the `...` "remainder" explanation in the description.